internal/iso/iso.go

```diff
@@ -21,7 +23,7 @@ import (
 )
 
 const (
-	defaultConsoles = "console=ttyS1 console=ttyS1 console=ttyS0 console=ttyAMA0 console=ttyS1 console=tty0"
+	defaultConsoles = "console=ttyS0 console=ttyAMA0 console=tty1 console=tty0, console=ttyS1,115200"
```

Comment:
Why are we changing the defaults?

Reply:
i've just been playing with the order and what not. I will revert this before the merge.

Reply:
oh sorry, i wont be reverting. the original has duplicates and is missing some good defaults.